packages/orchestration/src/facade.js

```diff
@@ -1,21 +1,25 @@
 /** @file Orchestration service */
 
 import { E } from '@endo/far';
+import { M } from '@endo/patterns';
 import { prepareCosmosOrchestrationAccount } from './exos/cosmosOrchestrationAccount.js';
+import { CosmosChainInfoShape } from './typeGuards.js';
+import { wellKnownChainInfo } from './chain-info.js';
 
 /**
  * @import {Zone} from '@agoric/zone';
  * @import {TimerService} from '@agoric/time';
  * @import {LocalChain} from '@agoric/vats/src/localchain.js';
  * @import {Remote} from '@agoric/internal';
  * @import {OrchestrationService} from './service.js';
- * @import {Chain, ChainInfo, CosmosChainInfo, KnownChains, OrchestrationAccount, Orchestrator} from './types.js';
+ * @import {Chain, ChainInfo, CosmosChainInfo, OrchestrationAccount, Orchestrator} from './types.js';
  */
 
 /** @type {any} */
 const anyVal = null;
 
 // FIXME should be configurable
+// ??? could the localchain service offer this?
 const mockLocalChainInfo = {
```

Comment:
`mockX` looks like test code, not `src` code.

Reply:
it's not test code either; it's an incomplete implementation. just like all the "'fixme'" strings we've had to date

Reply:
I've put issues at all the fixmes to be sure they'll be address